API/ev3_sensor.c

```diff
@@ -390,6 +390,67 @@ int ReadSensor(int sensorPort)
 	return *((DATA16*)data);
 }
 
+// gives the raw values of rgb, from 0 to 1023
+int ReadSensorRed(int sensorPort)
```

Comment:
Hi,

firstly thanks for the contribution and sorry for the delay.
I don't think `ReadSensorRed` is a good idea, because all other sensors go through `ReadSensor`. Could you add `COL_COLOR_{RED,GREEN,BLUE}` to `ReadSensor` and have them do the same you are doing here?

Also sorry for the garbled pull request output. It would be best if you make a new branch based on current master and `git push --force` it on top of your light-sensor branch.

Thanks!